src/Cli/Command/RequestCommand.php

```diff
@@ -89,9 +89,19 @@ protected function execute(InputInterface $input, OutputInterface $output)
 
         // Certificate renewal
         if ($this->hasValidCertificate($domain, $alternativeNames)) {
+            $this->getCliLogger()->debug('Certificate found, executing renewal', [
```

Comment:
a lot of calls to `$this->getCliLogger()->...` may be optimize with a `$logger =  $this->getCliLogger()` on top of the method.